runtime/runtimeUtil.py

```diff
@@ -73,6 +71,8 @@ class HIBIKE_RESPONSE(Enum):
 class ANSIBLE_COMMANDS(Enum):
     STUDENT_UPLOAD = "student_upload"
     CONSOLE        = "console"
+    TIMESTAMP_UP = "timestamp up"
```

Comment:
I think it would be worth it to copy the original strings informing future viewers what these actually mean.